wiki/osu!_rankings_history/en.md

```diff
@@ -0,0 +1,331 @@
+---
+tags:
+  - history
+  - leaderboard
+  - rank 1
+  - rank #1
+  - rankings
+  - top player
+---
+
+<!-- Maintainers Note: Please refer to the original pull request for more info on sources at https://github.com/ppy/osu-wiki/pull/4720 -->
+
+# osu! Rankings History
+
+The following is a list of players who are recognized to have reached Rank #1 in the [osu! Global Rankings](https://osu.ppy.sh/rankings/osu/performance).
+
+Over the years the osu! global rankings has used three different systems for ranking players:
+
+- [Ranked Score](/wiki/score#ranked-score) (July 2007 – 24 July 2012)
+- [ppv1](/wiki/Performance_points/ppv1) (24 July 2012 – 27 January 2014)
+- [ppv2](/wiki/Performance_points) (27 January 2014 – Present)
+
+Because of this, this list will only include players who reached Rank #1 on the main leaderboard that was used at the time of reaching said rank. *Italicized* dates and numbers indicate that the corresponding player are continuing their reign as osu!'s Rank #1 player either from the previous year or onto the following year.
+
+## 2007
+
+*Note: During the period where Ranked Score was used, Rank #1 would change frequently, thus the earlier listings may use months where exact dates aren't available.*
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ![][flag_US] [Ivalset](https://osu.ppy.sh/users/827) | 2007-10-11 | 2007-10-15 | [\[1\]](https://web.archive.org/web/20071011215722/http://osu.ppy.sh/index.php?p=player) |
+| ![][flag_AU] [eyup](https://osu.ppy.sh/users/88) | 2007-10-15 | 2007-10-24 | [\[1\]](https://osu.ppy.sh/community/forums/topics/342) |
+| ![][flag_US] [Ivalset](https://osu.ppy.sh/users/827) | 2007-10-24 | 2007-11-17 | [\[1\]](https://web.archive.org/web/20071024063540/http://osu.ppy.sh/index.php?p=player) |
+| ![][flag_NZ] [Echo](https://osu.ppy.sh/users/431) | 2007-11-17 | 2007-11-23 | [\[1\]](https://osu.ppy.sh/community/forums/topics/963) |
+| ![][flag_US] [Ivalset](https://osu.ppy.sh/users/827) | 2007-11-23 | 2007-12-24 | [\[1\]](https://web.archive.org/web/20071024063540/http://osu.ppy.sh/index.php?p=player) |
+| ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) | 2007-12-24 | *2008-09-07* | [\[1\]](https://web.archive.org/web/20071224062629/http://osu.ppy.sh/index.php?p=player) |
```

Comment:
is there a reason why there is only one bolded date here?

Reply:
`*Italicized* dates and numbers indicate that the corresponding player is continuing their reign as osu!'s Rank #1 player either from the previous year or into the following year.`

Since Ivalset isn't carrying it over from a previous year, It didn't make sense (to me) for it to be italicized.

Reply:
totally forgot abt that, my bad